battery

```diff
@@ -122,7 +131,7 @@ apply_colors() {
         fi
 
     # Red
-    elif [[ $BATT_PCT -lt 25 ]]; then
+    elif [[ "$BATT_PCT" -lt 25 ]]; then
```

Comment:
same question about strings applies to several places of this commit. It looks less clean, so I'm wondering if we really need this?

Reply:
Same as with the case block above. You are correct in that double brackets syntax doesn't require it, but it's a matter of style and choosing the safe side.